Source/WebCore/dom/Document.h

```diff
@@ -1380,6 +1383,8 @@ class Document
     void queueTaskToDispatchEventOnWindow(TaskSource, Ref<Event>&&);
     void dispatchPageshowEvent(PageshowEventPersistence);
     void dispatchPagehideEvent(PageshowEventPersistence);
+    void dispatchPageSwapEvent(bool canTriggerCrossDocumentViewTransition);
```

Comment:
nit: consistency with casing would be nice here

`canTriggerCrossDocumentViewTransition` could also be an enum class